app/views/site/_reviewer_access_fields.html.erb

```diff
@@ -0,0 +1,60 @@
+<div class="well well-sm" id="reviewer-access-controls">
+  <div class="form-group row">
+    <div class="col-md-2">
+      <%= f.label :expires_at, 'Expiration Date' %><br />
+      <%= f.date_field :expires_at, class: 'form-control' %>
+    </div>
+    <div class="col-md-2">
+      <%= label_tag :reset, 'Reset Access Credentials?' %><br />
+      <div class="btn-group btn-toggle" data-toggle="buttons">
+        <label class="btn btn-default">
+          <%= radio_button_tag 'reviewer_opts[reset]', :yes, false %> Yes
+        </label>
+        <label class="btn btn-default active">
+          <%= radio_button_tag 'reviewer_opts[reset]', :no, true %> No
+        </label>
+      </div>
+    </div>
+    <div class="col-md-2">
+      <% access_enabled = f.object.persisted? %>
+      <%= label_tag :_destroy, 'Access Enabled?' %><br />
+      <div class="btn-group btn-toggle" data-toggle="buttons">
+        <label class="btn btn-default <%= access_enabled ? 'active' : nil %>">
+          <%= radio_button_tag 'reviewer_opts[enable]', :yes, access_enabled, disabled: !access_enabled %> Yes
+        </label>
+        <label class="btn btn-default <%= access_enabled ? nil : 'active' %>">
+          <%= radio_button_tag 'reviewer_opts[enable]', :no, !access_enabled, disabled: !access_enabled %> No
+        </label>
+      </div>
+    </div>
+    <div class="col-md-6 left-border-0-5-gray">
+      <h4>Current Access Credentials <button class="btn btn-default btn-sm btn-copy" data-toggle="tooltip" title=""
+                                             data-original-title="Copy to clipboard"><i class="far fa-copy"></i></button>
+      </h4>
+      <% if access_enabled %>
+        <p id='access-credentials'>
+          <strong>Accession:</strong> <%= @study.accession %><br />
+          <strong>URL:</strong>&nbsp;<%= link_to reviewer_access_url(access_code: f.object.access_code),
+                                                 reviewer_access_url(access_code: f.object.access_code) %><br />
+          <strong>Pin:</strong>&nbsp;<%= f.object.pin %>
```

Comment:
We should mask the PIN display and only show it for a short period of time when clicking on a "show" button. Doing so minimizes the risk of a rogue script or browser extension "grabbing" the PIN from the raw HTML..

Reply:
I'll look into doing that - great suggestion.